README.md

```diff
@@ -1,33 +1,56 @@
 # WP DETECT
 A WordPress detection tool, detects if a website is running WordPress. wpdetect is a great tool when you just want to check WordPress' presence but do not want to scan the site for vulnerabilities or issues.  
 ### Installation
+
+#### Installing from pip
 You can install wpdetect using pip,
 ```sh
 pip install wpdetect
 ```
 wpdetect requires Python 3 or above to run. If you have Python 2 installed too, make sure to use the right pip.
 
+
 ### Usage
 Syntax
+
+
+```sh
+wpdetect --url https://www.malcare.com/
```

Comment:
Instead of this, please use https://wordpress.org

Reply:
Implemented